recipes/polystest/meta.yaml

```diff
@@ -6,12 +6,12 @@ package:
   version: {{ version }}
 
 build:
-  number: 0
+  number: 1
   noarch: generic
 
 source:
-  url: https://bitbucket.org/veitveit/polystest/get/release-1.2.tar.gz
-  sha256: bf9078a517d69c01c6fc551ac271fc3cc19c35a62670daca5e8e077399ab13a5
+  url: https://bitbucket.org/veitveit/polystest/get/release-1.2.2.tar.gz
```

Comment:
Your PR title states version 1.2.1, this line states version 1.2.2, line two states 1.2.

Please set the correct version in line two, then change this line to `url: https://bitbucket.org/veitveit/polystest/get/release-{{ version }}.tar.gz`

Reply:
Sorry about the confusion. This should be fixed now.